shared/src/packages/users/libs/types/user-details-dto.type.ts

```diff
@@ -2,6 +2,10 @@ type UserDetailsDto = {
   id: number;
   firstName: string;
   lastName: string;
+  avatarUrl: string | null;
+  avatar?: {
+    url: string | null;
+  };
```

Comment:
Why do we need to duplicate it? Can't we just have a single avatarUrl field?

Reply:
If I remove
```
 avatar?: {
    url: string | null;
  };
```
I will get an error that the property 'avatar' does not exist on type 'UserDetailsResponseDto' in the article entity. Is there a workaround for this?